src/Vehicle/Vehicle.h

```diff
@@ -17,7 +17,13 @@ namespace LockFreeDispatch {
         float curWaterVolumeLitres;
         float maxWaterVolumeLitres;
         VehicleStatus curVehicleStatus;
+        float workFactor;
```

Comment:
Consider whether order is important for alignment and cache utilisation. (If not, might be best to alphabetise.)